js/src/ui/publishBox.js

```diff
@@ -86,6 +119,18 @@ var imageScoreClass = "image yoast-logo svg";
 		if ( wpseoPostScraperL10n.keywordAnalysisActive === "1" ) {
 			createScoresInPublishBox( "keyword", notAvailableStatus );
 		}
+
+		$( "#content-score" ).click( function( event ) {
```

Comment:
Just realized this makes the whole `<div>` element clickable (same for the SEO div). Clicking on the empty area close to the link will unexpectedly trigger the scrolling. We should target only the link, as we don't want to use clickable divs. Besides the fact it's not semantic and problematic for accessibility, this caught me when trying to select the link text with the mouse: I just wanted to select the text, and it triggered the scrolling.